src/lib/widget/modal.svelte

```diff
@@ -25,47 +21,29 @@
     onMount(fn: (dialog: HTMLDialogElement, form: HTMLFormElement) => void) {
       callMounted = (dialog, form) => fn(dialog, form);
     },
-    open(options?: JourneyOptions): void {
-      // If journey does not have a step, start the journey
-      if (!get(api.getJourneyStore()).step) {
-        journey.start(options);
-      }
+    open() {
```

Comment:
The more I think about the fact that modal.open() and journey.start() are no longer linked, the more I think this is beneficial rather than a drawback. There may well be some scenario where the developer wants to open the modal but allow the user to select something within that modal, before starting the journey - e.g. something that might guide which journey actually gets launched. I'm thinking of when we developers are able to drop their own code into the widget. I think this isn't such a bad thing, even if we factor in the extra line of code to start the journey.